buildinglocally.md

```diff
@@ -359,7 +358,7 @@ mvn clean install -DskipTests -Dmaven.javadoc.skip=true
 # or cross-build across Scala versions
 # ./buildmultiplescalaversions.sh clean install -DskipTests -Dmaven.javadoc.skip=true
 ## If you skipped CUDA you may need to add 
-## -pl '!:deeplearning4j-cuda-8.0' 
+## -pl '!:deeplearning4j-cuda-9.0'
```

Comment:
I've found this is cuda version dependent. What you want to add to be independent of that version is this:

Reply:
- Added ` -pl '!./deeplearning4j-cuda/'` instead of the above.
- Also made cross-building across scala versions the default for dl4j.